<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui</class>
 <widget class="QMainWindow" name="QtGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>28</y>
      <width>516</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="3" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushD">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushC">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push2">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push0">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push4">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushF">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushB">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushE">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>E</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push1">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push6">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push3">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push5">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push8">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push7">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="push9">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushA">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_ROM"/>
    <addaction name="actionClose_ROM"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_ROM">
   <property name="text">
    <string>Open ROM</string>
   </property>
  </action>
  <action name="actionClose_ROM">
   <property name="text">
    <string>Close ROM</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
